### DISCUSSION TITLE
Discussions Topics for Dec 6, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by clediggins-usds** [CDCgov/prime-reportstream#3302] [Show and tell]
Discussion topics for Dec 6

**Comment by clediggins-usds** [comment]
Batch process change, cron job per receiver, update how we use batch queue. Warehouse metaphor

**Comment by cwinters-usds** [comment]
Initial draft of next steps for smaller and more frequent deployments.

**Comment by jimduff-usds** [comment]
I'd like to review the new features soon to available for managing ReportStream Filters.   If needed, this could wait a week, as the feature is not yet in Production.

**Comment by jimduff-usds** [comment]
I could also talk about that weird `Value cannot be null. (Parameter 'provider')` bug I was seeing, if you think that's helpful.

**Comment by carlosfelix2** [comment]
License change for Docker Desktop

**Comment by brick-green** [comment]
OTC results need global default values for various fields depending on the type (at-home, prescription, tele-proctored). Where should that work be completed? Should there be separate schemas for the three types with the default values? Should those be applied prior to internal report creation? Maybe instead of schemas we mappers then apply to sender schemas?

**Comment by RickHawesUSDS** [comment]
I'd like to show the initial design for the ACK and NACK Data Flow and architecture changes

**Comment by sean-usds** [comment]
Creating a Docker glossary to help with debugging
